src/components/mods/MOptions.ts

```diff
@@ -0,0 +1,39 @@
+export class MOptions {
```

Comment:
IMO it's much better to explicitly name a class what it does without abbreviations. Saving the two characters for `od` here means newcomers have to learn what this abbreviation means... which can be very confusing.